src/cli/utils/groups.go

```diff
@@ -172,11 +179,13 @@ func AddGroupsFilter(
 // inclusions for Group commands.
 func IncludeGroupsRestoreDataSelectors(ctx context.Context, opts GroupsOpts) *selectors.GroupsRestore {
 	var (
-		groups      = opts.Groups
-		lfp, lfn    = len(opts.FolderPath), len(opts.FileName)
-		llf, lli    = len(opts.ListFolder), len(opts.ListItem)
-		lpf, lpi    = len(opts.PageFolder), len(opts.Page)
-		lg, lch, lm = len(opts.Groups), len(opts.Channels), len(opts.Messages)
+		groups     = opts.Groups
+		lfp, lfn   = len(opts.FolderPath), len(opts.FileName)
+		llf, lli   = len(opts.ListFolder), len(opts.ListItem)
+		lpf, lpi   = len(opts.PageFolder), len(opts.Page)
+		lg         = len(opts.Groups)
+		lch, lm    = len(opts.Channels), len(opts.Messages)
+		lcon, lpst = len(opts.Conversations), len(opts.Posts)
```

Comment:
These acronyms are hard to follow through this func. This is one of the cases where I think we should just use `len(opts.Foo)` wherever they are being used. Otherwise it's easy to miss a letter and cause bugs. 

Also, the test (`TestIncludeGroupsRestoreDataSelectors`) only checks for include length. We should improve it to match against expected selectors.

Reply:
Yeah, they're getting out of hand, aren't they?  I'll clean that up.